src/services/anchor-service.ts

```diff
@@ -180,6 +180,7 @@ export default class AnchorService {
       logger.debug(`Successfully anchored CID ${candidate.cid.toString()} for document ${candidate.document.id.toString()}`)
     }
     logger.imp(`Service successfully anchored ${anchors.length} CIDs.`);
+    process.exit()
```

Comment:
It feels like this should be happening at a higher layer.  There are other ways that `anchorRequests` can be called, including periodically on a schedule [when in `bundled` mode](https://github.com/ceramicnetwork/ceramic-anchor-service/blob/857a5bb93475a265b6dfaa12a8e7d440dac209e5/src/services/scheduler-service.ts#L34), or [according to an incoming request](https://github.com/ceramicnetwork/ceramic-anchor-service/blob/d224299195dbac599db000fdd80f0adbeb1523f8/src/controllers/anchor-controller.ts#L22) and this would cause both of those to terminate the process prematurely.

Really this feels like this should happen at the top level, [here](https://github.com/ceramicnetwork/ceramic-anchor-service/blob/6286ffd671ab5a7bf97cc592ec5a509b9032cc8f/src/app.ts#L58).  But maybe that's too late in the process that you're worried it won't stop the hang?  When you see the hanging processes, do you still see the `Ceramic Anchor Service started in anchor mode` [log message](https://github.com/ceramicnetwork/ceramic-anchor-service/blob/6286ffd671ab5a7bf97cc592ec5a509b9032cc8f/src/app.ts#L59)?

Reply:
I think the code you are pointing to is outdated but yeah you are right. Moving this to app.ts and will test in bundled mode too

Reply:
Woah weird, not sure how I wound up on that version